### DISCUSSION TITLE
Docs: Update step 5 for Tailscale integration

### DISCUSSION THREAD

**Discussion by garrett-ts** [authelia/authelia#5359] [Ideas]
Hi everyone!

Looking at the [Tailscale integration doc](https://www.authelia.com/integration/openid-connect/tailscale/) I noticed a small inaccuracy, specifically on step 5:

> Once you have the WebFinger reply set up and your [Authelia OpenID Connect Discovery endpoint](https://www.authelia.com/integration/openid-connect/introduction/#well-known-discovery-endpoints) is working (e.g. https://auth.example.com/.well-known/openid-configuration), you can sign up for a new Tailnet (**currently migration isn’t supported**) via the link: [Sign up with OIDC](https://login.tailscale.com/start/oidc) where you will see the following screen:

Migrating to custom OIDC _is_ possible with Tailscale, but can only be done if the tailnet is associated with a custom domain; it isn't possible for a public/shared domain (e.g. gmail.com). Our documentation for custom OIDC touches on migration here: https://tailscale.com/kb/1240/sso-custom-oidc/#migrate-an-existing-tailnet

We love seeing community guides that help people use their IdP with Tailscale, but I'd suggest adding a disclaimer on the page to always check the official documentation as well, in case something has changed.

**Comment by james-d-elliott** [comment]
> We love seeing community guides that help people use their IdP with Tailscale, but I'd suggest adding a disclaimer on the page to always check the official documentation as well, in case something has changed.

See the very first alert in the guide, and the references in the see also section. ;)

**Reply by james-d-elliott** [reply]
I wrote this as I flew out the door to work this morning, and didn't get a chance to convey everything I was thinking. Thanks for reaching out, by the way. We'll correct that specific nuance. The OIDC guides are mostly all community contributed, but we always push users towards looking at client docs where appropriate as we can only support specifically our side of the flow naturally.

It'd be great if we can get ourselves listed in the tailscale docs as well.

**Reply by catzkorn** [reply]
We've been working on that today and expect it to be live in our custom OIDC docs tomorrow!

**Reply by james-d-elliott** [reply]
Nice! Also for reference 6b04fe2652d1f7a3b42fcb4907319ea99d8139d4 fixed this. I cherry picked it into production docs.

**Reply by garrett-ts** [reply]
Thanks for the help James! Apologies for missing the specifics in those sections like you recommended. The messaging is perfectly clear. Cheers!

**Comment by catzkorn** [comment]
Hi! Update from us at Tailscale 😄 

We announced via our [changelog](https://tailscale.com/changelog/) support for Authelia & added the links to your doc in our [custom OIDC doc](https://tailscale.com/kb/1240/sso-custom-oidc/#additional-provider-configurations)

**Reply by james-d-elliott** [reply]
Nice!
